test-defaults.ini

```diff
@@ -6,6 +6,9 @@ ws.auth_required = False
 
 is_test_running = True
 
+# Prevent plugins from loading during tests
+plugins_dir = '%(root)s/test_plugins'
```

Comment:
This is a non-existent directory to prevent sideboard from attempting to load plugins during tests.